doc-content/enterprise-only/ai/ai-gsg-proc.adoc

```diff
@@ -0,0 +1,178 @@
+. Add the *Risk Predictor* `dtree_risk_predictor.pmml` model.
++
+image:ai/include-model.png[]
++
+.. In the *Included Models* window of the DMN editor, click *Include Model*.
+.. In the *Include Model* dialog, select `dtree_risk_predictor.pmml` from the *Models* menu.
+.. Enter `Risk Predictor` in the *Provide a unique name* box and click *OK*.
+
+. Create the *Risk Predictor* Business Knowledge Model (BKM) node with the *Risk Predictor*  and *DecisionTreeClassifier* model.
++
+image:ai/risk-predictor-function.png[]
+
+.. In the *Model* window of the DMN editor, drag a BKM node to the palette.
++
+image:ai/bkm.png[]
+.. Rename the node *Risk Predictor*.
+.. Click the edit icon located below the trash can icon on the left side of the node.
++
+image:ai/risk-predictor-node.png[]
+.. Click *F* in the *Risk Predictor* box and select *PMML* from the *Select Function Kind* menu. The *F* changes to *P*.
+.. Double-click the *First select PMML document* box and select *Risk Predictor*.
+.. Double-click the *Second select PMML model* box and select *DecisionTreeClassifier*.
+.. To return to the DMN editor palette,  click *Back to Dispute Transaction Check*.
+
+. Create the *Transaction* input data node with the data type *tTransaction*.
++
+image:ai/risk-transaction.png[]
+
+.. In the *Model* window of the DMN editor, drag an input data node to the palette.
++
+image:ai/input-node.png[]
+
+.. Rename the node *Transaction*.
+.. Select the node then click the properties pencil icon in the upper right corner of the window.
+.. In the *Properties* panel, select *Information Item -> Data type -> tTransaction* then close the panel.
+
+. Create the *Transaction Dispute risk* decision node and add the *Transaction* node for data input and the *Risk Predictor* node for the function.
+.. In the *Model* window of the DMN editor, drag a decision data node to the palette.
++
+image:ai/decision-node.png[]
+
+.. Rename the node *Transaction Dispute Risk*.
+.. Select the *Risk Predictor* node and drag the arrow from the top right of the node to the *Transaction Dispute Risk* node.
+.. Select the *Transaction* node and drag the arrow from the top right of the node to the *Transaction Dispute Risk* node.
+
+. In the *Transaction Dispute Risk* node, create the *Risk predictor* invocation function.
++
+image:ai/transaction-dispute-risk.png[]
+
+.. Select the *Transaction Dispute Risk* node and click the edit icon on the left side of the node.
+.. Click *Select expression* and select *Invocation* from the menu.
+.. Enter *Risk Predictor* in the *Enter function* box.
+.. Click *P1*.
+.. In the *Edit Parameter* dialog, enter `amount` in the *Name* box, select *number* from the *Data Type* menu, and press the Enter key.
+.. Click *Select expression* and select *Literal expression* from the menu.
+.. Enter `Transaction.transaction_amount` in the box next to *amount*.
+.. Right click on *1* and select *Insert below*. The *Edit Parameter* dialog opens.
```

Comment:
Please change "Right click" to Right-click".

Reply:
Done